src/docs/truffle/getting-started/installation.md

```diff
@@ -18,3 +18,133 @@ Truffle also requires that you have a running Ethereum client which supports the
 ## Recommendations for Windows
 
 If you're running Truffle on Windows, you may encounter some naming conflicts that could prevent Truffle from executing properly. Please see [the section on resolving naming conflicts](/docs/advanced/configuration#resolving-naming-conflicts-on-windows) for solutions.
+
+## Recommendations for Linux
+
+If you're running Truffle on Linux or macOS, you should not install it with `sudo`, otherwise you might encounter some permission issues. It is recommended to install Truffle as a normal user. Before we can install Truffle as a normal user, we have to clean-up the system. It is also recommended to install Node.js and NPM with [NVM (Node Version Manager)](https://github.com/creationix/nvm]. An advantage of NVM is that you can run different Node versions on the same machine.
+
+### Cleanup node and npm
+#### Remove all packages which have been installed with `sudo npm install -g`
+
+First we want to find out which packages have been installed with `sudo npm install -g`.
+Get a list of all installed packages:
+
+```bash
+$ sudo npm list -g --depth=0
```

Comment:
Also, since we recommend not using `sudo`, can we also remove them from all the `npm` commands below? This would be nice as I encounter a lot of users having permissions issues when doing this.